### DISCUSSION TITLE
What is the difference between set vs setState?

### DISCUSSION THREAD

**Discussion by soyoonJ** [pmndrs/zustand#2698] [Q&A] [answered]
https://zustand.docs.pmnd.rs/guides/flux-inspired-practice#use-set-/-setstate-to-update-the-store

According to Docs, it saids 'Always use set (or setState) to perform updates to your store.'

But I can't find difference between set and setState. Why there are two ways to update store(set, setState)? Is there any difference or reason?

**Comment by dbritto-dev** [comment, marked as answer]
@soyoonJ there's no difference between `set` and `setState` they're available on different phases, `set` is available at store creation and `setState` after store creation

**Reply by soyoonJ** [reply]
thank you for your reply!
